test/functional/page_objects/visualize_page.js

```diff
@@ -748,7 +748,7 @@ export function VisualizePageProvider({ getService, getPageObjects, updateBaseli
       }
     }
 
-    async saveVisualization(vizName, { saveAsNew = false } = {}) {
+    async saveVisualization(vizName, { saveAsNew = false, waitForSaveComplete = true } = {}) {
```

Comment:
I'd rather we make this consistent with the recent changes in dashboard_page where the save method always;
1. closes the toast and gets the text from it
2. then waits for loading to finish
3. then returns the toast message in case any test wants to check it

The problem with waitUntilLoadingHasFinished is that it can take more than 5 seconds and then the toast is closed and a test waiting to find it fails.